### DISCUSSION TITLE
Changedetection viewport to small

### DISCUSSION THREAD

**Discussion by Lockie85** [browserless/chrome#4374] [Q&A] [answered]
Hey,

I'm using changedetection.io and I've installed it on my Proxmox server via this script:

https://tteck.github.io/Proxmox/
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/changedetection.sh)"

When accessing changedetectionand its Visual Selector tool which utilises (Ibelieve) browserless. The view is to small. I've defined a larger width and height within file:
`/etc/systemd/system/changedetection.service`

Like so:
`Environment=PLAYWRIGHT_DRIVER_URL=ws://localhost:3000/chrome?launch={"defaultViewport":{"height":1080,"width":1920},"headless":false,"stealth":true}&blockAds=true`

However, no luck, the view is still small. Can someone help me figure out what might be going wrong?

Many Thanks

**Comment by Lockie85** [comment]
Anyone? :(

**Comment by joelgriffith** [comment, marked as answer]
Not sure we can directly answer as it's up to how changedetection.io runs things on their end :/
